lib/mayaUsd/fileio/shading/shadingModeExporterContext.cpp

```diff
@@ -492,8 +492,35 @@ UsdMayaShadingModeExportContext::MakeStandardMaterialPrim(
                 UsdShadeMaterialBindingAPI bindingAPI =
                     UsdMayaTranslatorUtil::GetAPISchemaForAuthoring<
                         UsdShadeMaterialBindingAPI>(boundPrim);
-                UsdGeomSubset faceSubset = bindingAPI.CreateMaterialBindSubset(
-                    /* subsetName */ TfToken(materialName),
+
+                UsdGeomSubset faceSubset;
+                TfToken materialNameToken = TfToken(materialName);
+
+                // Try to re-use existing subset if any:
+                for (auto subset : bindingAPI.GetMaterialBindSubsets()) {
+                    TfToken elementType;
+                    if (subset.GetPrim().GetName() == materialNameToken
+                        && subset.GetElementTypeAttr().Get(&elementType)
+                        && elementType == UsdGeomTokens->face) {
+                        faceSubset = subset;
+                        break;
+                    }
+                }
+
+                if (faceSubset) {
+                    // Update and continue:
+                    VtIntArray mergedIndices;
+                    UsdAttribute indicesAttribute = faceSubset.GetIndicesAttr();
+                    indicesAttribute.Get(&mergedIndices, UsdTimeCode::Default());
+                    std::set<int> uniqueIndices(mergedIndices.cbegin(), mergedIndices.cend());
+                    uniqueIndices.insert(faceIndices.cbegin(), faceIndices.cend());
+                    mergedIndices.assign(uniqueIndices.cbegin(), uniqueIndices.cend());
+                    indicesAttribute.Set(mergedIndices, UsdTimeCode::Default());
```

Comment:
I might leave off the `time` parameter to the `UsdAttribute` `Get()` and `Set()` calls. It's optional and what we're passing here matches the default, but their presence suggests that we're intending to do something specific related to time, which I don't think we are. I think it would be fairly weird if the binding subsets were time varying, if that's even supported at all in Maya.